src/lib/pages/workflow-workers.svelte

```diff
@@ -1,13 +1,28 @@
 <script lang="ts">
+  import { page } from '$app/stores';
+
   import WorkersList from '$lib/components/workers-list.svelte';
+  import { getWorkerTaskReachability } from '$lib/services/pollers-service';
   import { workflowRun } from '$lib/stores/workflow-run';
+  import {
+    getUniqueBuildIdsFromPollers,
+    pollerHasVersioning,
+  } from '$lib/utilities/task-queue-compatibility';
+
+  let { namespace } = $page.params;
+  $: ({ workers, workflow, compatibility } = $workflowRun);
 
-  $: ({ workers, workflow, compatibility, reachability } = $workflowRun);
+  $: taskQueue = workflow?.taskQueue;
+  $: versioningEnabled = pollerHasVersioning(workers.pollers);
 </script>
 
-<WorkersList
-  taskQueue={workflow?.taskQueue}
-  {workers}
-  {compatibility}
-  {reachability}
-/>
+{#if versioningEnabled}
+  {@const buildIds = getUniqueBuildIdsFromPollers(workers.pollers)}
+  {#await getWorkerTaskReachability( { namespace, buildIds, taskQueue }, ) then reachability}
```

Comment:
Do we want some kind of loading state here? Maybe a skeleton table?

Reply:
👍 good call, adding it